test/typescript/app.ts

```diff
@@ -45,7 +45,7 @@ export function fetchApp(organization: Organization) {
         })
 
         test('Find all Apps', done => {
-            organization.app().findAll()
+            organization.marketplace().findAllApps()
```

Comment:
Please review the naming convention. Should it be find() or findAll()? It's worth noting that find() is generally associated with retrieving all available content.